tests/test_resolver.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 The Foundry Visionmongers Ltd
+
+# pylint: disable=no-self-use
+# pylint: disable=invalid-name,redefined-outer-name
+# pylint: disable=missing-class-docstring,missing-function-docstring
+
+import pytest
+import os
+
+# This environment var must be set before the usd imports.
+os.environ["TF_DEBUG"] = "OPENASSETIO_RESOLVER"
+from pxr import Plug, Usd
+
+
+@pytest.fixture(autouse=True)
+def register_plugin():
+    # plugInfo.json file for the example resolver should be installed
+    # in the resources folder under the dist directory.
+    Plug.Registry().RegisterPlugins(
+        os.path.join(os.getcwd(), "../build/dist/resources")
```

Comment:
This code is just a test convenience to avoid having to set `PXR_PLUGINPATH_NAME` before you run the test. Do you think just making "Set `PXR_PLUGINPATH_NAME` to point to your installed plugInfo.json, wherever it may be" a neccesary manual step is the way to go with this then?

Reply:
Sorry I'd missed this, but its good if you can also avoid `cwd`, as `pytest` isn't always run at the root. We do [this](https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/blob/795387017e6775d933ea701cccc3139c53127a69/tests/conftest.py#L39) in other projects.

Reply:
Have just made it neccesary to set the environment variable manually for the tests.

[1ecc802](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3/commits/1ecc8024fd445435905a2ee058675aad600a9c19)
[caa5e39](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3/commits/caa5e39f1191fccf2e3b5d781fd0418b1beaff9b)

Reply:
Thanks, we can tweak this in the coming work if we need to.